documentation/staging/content/userguide/base-images/access-images.md

```diff
@@ -0,0 +1,115 @@
+  You need to do this only once for a particular image.
+  See [Obtain images from the Oracle Container Registry](#obtain-images-from-the-oracle-container-registry).
+
+For more information about creating Kubernetes Secrets for accessing
+the registry, see the Kubernetes documentation,
+[Pull an image from a private registry](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/).
+
+#### Option 2: Store images in a central registry and set up a Kubernetes service account with image pull secrets in each domain namespace
+
+An option for accessing an image that is stored in a private registry
+is to set up the Kubernetes `ServiceAccount` in the namespace running the
+WebLogic domain with a set of image pull secrets thus avoiding the need to
+set `imagePullSecrets` for each `Domain` resource being created (because each resource
+instance represents a WebLogic domain that the operator is managing):
+
+- Create a Kubernetes `docker-registry` secret as shown
+  in [Option 1](#option-1-store-images-in-a-central-registry-and-set-up-image-pull-secrets-on-each-domain-resource).
+
+- Modify the `ServiceAccount` that is in the same namespace
+  as your domain resources to include this image pull secret:
+
+  ```shell
+  $ kubectl patch serviceaccount default -n domain1-ns \
+  -p '{"imagePullSecrets": [{"name": "my-registry-pull-secret"}]}'
+  ```
+
+  Note that this patch command entirely replaces the current list of
+  image pull secrets (if any). To include multiple secrets, use
+  the following format:
+  `-p '{"imagePullSecrets": [{"name": "my-registry-pull-secret"}, {"name": "my-registry-pull-secret2"}]}'`.
+
+For more information about updating a Kubernetes `ServiceAccount`
+for accessing the registry, see the Kubernetes documentation,
+[Configure service accounts for pods](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#add-image-pull-secrets-to-a-service-account).
+
+#### Option 3: Manually place images on Kubernetes cluster nodes
+
+Alternatively, it may be preferable to manually place an image
+on each Kubernetes worker node in your Kubernetes cluster in advance.
+
+For example, if the desired image is located in a Docker registry,
+then you can manually call `docker login` and `docker pull` on each
+worker node. For the steps with Oracle Container Registry images,
+see [Obtain images from the Oracle Container Registry](#obtain-images-from-the-oracle-container-registry).
+
+As another example,
+if the Docker image is located in a local Docker cache,
+then you can get an inventory of the cache by calling `docker images`:
+
+- Save the image to a TAR file: `docker save -o myimage.tar myimagerepo:myimagetag`.
+- Then, copy the TAR file to each node.
+- Call `docker load -o myimage.tar` on each node.
+
+If you choose this approach, then a Kubernetes Secret is not required
+and your domain resource `domain.spec.imagePullPolicy` must be set to `Never` or `IfNotPresent`.
```

Comment:
Hmm...  do we really need to even mention this manual distribution of images to node hack in this day and age?

Reply:
I suspect that customers who do this know why they selected it and don't need us to tell them. We've become more verbose over time because a surprising number of customers can't figure out why they are getting image pull backoff errors. I would lean toward agreeing that the comment about image pull secrets and registries is sufficient.

Reply:
@rjeberhard Need clarification here: should I remove this sentence, "If you choose this approach, then a Kubernetes Secret is not required and your domain resource `domain.spec.imagePullPolicy` must be set to `Never` or `IfNotPresent`"?

Reply:
I think that @robertpatrick is suggesting that the entire option 3 is unnecessary. The reason is that an extremely small number of customers do this and, for those that do, they are already needing to do this for all of their images and so don't need our help to understand how to do this specifically for the operator's image.

Reply:
okey doke. Got it. I will remove Option 3.